src/main/java/uk/gov/ch/service/capital/impl/StatementOfCapitalServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.ch.service.capital.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.StatementOfCapitalNotFoundException;
+import uk.gov.ch.model.capital.StatementOfCapital;
+import uk.gov.ch.repository.capital.StatementOfCapitalRepository;
+import uk.gov.ch.service.capital.StatementOfCapitalService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import java.util.List;
+
+@Service
+public class StatementOfCapitalServiceImpl implements StatementOfCapitalService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OracleQueryApplication.APPLICATION_NAME_SPACE);
+
+    @Autowired
+    private StatementOfCapitalRepository statementOfCapitalRepository;
+
+    @Override
+    public StatementOfCapital getStatementOfCapital(String incorporationNumber) throws StatementOfCapitalNotFoundException {
+        List<StatementOfCapital> statementOfCapitalList = statementOfCapitalRepository.getStatementOfCapital(incorporationNumber);
+        int size = statementOfCapitalList.size();
+        if (size == 1) {
+          return statementOfCapitalList.get(0);
+        } else {
+           LOGGER.error("Single result not found when getting statement of capital");
+           throw new StatementOfCapitalNotFoundException("Statement of capital results " + size);
```

Comment:
this condition would trigger if more than one were found.
Is the the best behaviour? I assume that we are only expecting 1 to be found here. In which case if there is more than 1, would it be better to end up throwing a status 500? As that is an unexpected scenario.
And if 0 found, then as it does here, return a 404

Reply:
Updated this.